quiz-bundle-spec.md

```diff
@@ -29,7 +29,25 @@ schema_version     | ✓        | integer    | Which version of the quiz bundle
 title              |          | dictionary | A locale dictionary of the quiz title, such as "My Awesome Quiz"
 passing_percentage | ✓        | integer    | The threshold grade that a student needs to achieve in order to count as "passing" the quiz.
 duration           |          | integer    | The default time a student is allotted for the quiz, in minutes. If unspecified, the quiz will not be timed.
-items              | ✓        | array      | An ordered array of `items` (see below for details) in this quiz - items will appear to students in this order
+sections           |          | array      | An ordered array of `sections` (see below for details) in this quiz; not allowed if `items` are specified; required if `items` are unspecified.
+items              |          | array      | An ordered array of `items` (see below for details) in a default section of this quiz; not allowed if `sections` are specified; required if `sections` are unspecified.
+
+### Sections
+Sections are groupings of related quiz items.
+
+```yml
+id: section-0
+name: Networking
+item_count: 2
+items: ...
+```
+
+attribute          | required | type       | notes
+-------------------| -------- | ---------- | -----------------------------------------
+id                 | ✓        | string     | A unique identifier for this section
+name               |          | string     | A name for this section
+item_count         |          |            | The number of items to select from this section when composing the quiz displayed to the student. If unspecified, defaults to the total number of `items` specified below.
+items              | ✓        | array      | An ordered array of `items` (see below for details) in this section
```

Comment:
please add a note about using `item_count` for item bank functionality

Reply:
Done.  Added item bank wording to section description as well.